src/accounts/get-account-transfer-history.js

```diff
@@ -1,7 +1,37 @@
 "use strict";
 
+/**
+ * @typedef {Object} AccountTransfer
+ * @property {string} transactionHash Hash returned by token transfer.
+ * @property {number} logIndex Number of the log index position in the Ethereum block containing the transfer.
+ * @property {number} creationBlockNumber Number of the Ethereum block containing the transfer.
+ * @property {string} blockHash Hash of the Ethereum block containing the transfer.
+ * @property {number} creationTime Timestamp, in seconds, when the Ethereum block containing the transfer was created.
+ * @property {string} sender Ethereum address of the token sender.
+ * @property {string} recipient Ethereum address of the token recipient.
```

Comment:
Worth mentioning that it `sender` or `recipient` could be null, indicating a burn or a mint?

Reply:
Ah, good point.  Yeah, I'll add that.